src/client/caseCorrect.js

```diff
@@ -1,7 +1,8 @@
 var yggdrasil = require('yggdrasil')({});
 var UUID = require('uuid-1345');
 
-module.exports = function(client, options) {
+module.exports = function(client) {
+  var options = client.options;
```

Comment:
Why client.options ? The `client, options` pattern is consistent with what mineflayer is doing, and it seems to makes sense. (It also allows destructuring : for example https://github.com/PrismarineJS/flying-squid/blob/master/src/lib/plugins/world.js#L10 )